ui/styl/partials/navigation-bar.styl

```diff
@@ -57,22 +57,30 @@ header
       color $main-nav-inactive
       list-style-type none
 
-      path
+      &:not(:first-child) path
         fill $main-nav-inactive
 
       &.active, &.active:hover
         color $main-nav-active-font
         background-color $main-nav-active-bg
 
-        path
+        &:not(:first-child) path
           fill $main-nav-active-icon
 
-      &:hover
+      &:hover:not(:first-child)
         color $main-nav-hover
 
         path
           fill $main-nav-hover
 
+      &:first-child
```

Comment:
This is special styling for the cockroach icon?  It would likely be more clear to give it a special class, rather than use "first-child" here.